.travis.yml

```diff
@@ -11,8 +11,4 @@ install:
   - travis_retry mvn dependency:resolve
 
 script:
-  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar
-
-cache:
-  directories:
-    - "$HOME/.m2"
```

Comment:
why was this fialing?

Reply:
I don't know. I returned it back and it works now.